x-pack/plugins/infra/public/components/asset_details/components/alerts_tooltip_content.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ALERTS_DOC_HREF } from '../../../common/alerts/constants';
+import { LinkToAlertsHomePage } from '../links/link_to_alerts_page';
+
+export const AlertsTooltipContent = React.memo(() => {
+  const onClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    e.stopPropagation();
+  };
+
+  return (
+    <EuiText size="xs" onClick={onClick}>
+      <FormattedMessage
+        id="xpack.infra.assetDetails.alerts.tooltip.alertsLabel"
+        defaultMessage="Showing alerts for this host. You can create and manage alerts in {alerts}"
+        values={{
+          alerts: <LinkToAlertsHomePage />,
+        }}
+      />
+      <FormattedMessage
```

Comment:
Should we add a blank line between these 2? I think it would make it consistent with the tooltips in the table:

<img width="507" alt="image" src="https://github.com/elastic/kibana/assets/2767137/742c0f5a-9be2-4e53-a4b5-4e155e7611a4">

Reply:
I wrapped them in paragraphs like the other tooltips 👍